config/shopr.php

```diff
@@ -10,6 +10,14 @@
         'order-confirmation' => '',
     ],
 
+    /*
+     * Create a easier way to apply custom config upon base models.
```

Comment:
I'd change this to:
```
/*
 * The database models. You may swap these with your own models, just make sure they extend the
 * corresponding package model rather than the standard Eloquent model.
 */
```